images/phpstorm/README.md

```diff
@@ -0,0 +1,13 @@
+# Enterprise PhpStorm
+
+[![Docker Pulls](https://img.shields.io/docker/pulls/codercom/enterprise-phpstorm?label=codercom%2Fenterprise-phpstorm)](https://hub.docker.com/r/codercom/enterprise-phpstorm)
+
+## Description
+
+Wraps [enterprise-multieditor](../multieditor/README.md) with PhpStorm
+installation.
+
+## How To Use It
+
+This image is ready for direct use within Coder Enterprise. Environments created
```

Comment:
```suggestion
This image is ready for direct use within Coder Enterprise. Workspaces created
```

Reply:
all of these images need some TLC, there's typos like that everywhere - I don't think we use the name "Coder Enterprise" anymore, either

Reply:
Agreed. Should we just be consistent for now and ditch this suggestion?